server/schemas/internel/v1/schema.proto

```
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package reearth.cms.v1;
```

Buf (1.47.2)

Files with package "reearth.cms.v1" must be within a directory "reearth/cms/v1" relative to root but were in directory "server/schemas/internel/v1".

(PACKAGE_DIRECTORY_MATCH)